fastlane/lib/fastlane/server/socket_server.rb

```diff
@@ -134,8 +136,8 @@ def send_response(json)
       return COMMAND_EXECUTION_STATE[:ready]
     end
 
-    def listen
-      @server = TCPServer.open('localhost', 2000) # Socket to listen on port 2000
+    def listen(port)
```

Comment:
We can probably remove the `@port` as a parameter here since its already an instance variable? 🙃